### ISSUE TITLE
TypeScript paths alias does't work with the command "nest build" as sequential path segment separation characters on Windows

### ISSUE THREAD

**Issue by iWinston** [nestjs/nest-cli#513]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

![image](https://user-images.githubusercontent.com/23229557/71554832-83d11980-2a5f-11ea-93e5-ab633e49aa13.png)

## Expected behavior

Paths alias should work well after compiled
## Minimal reproduction of the problem with instructions

1. Running "nest new nest-example", then cd nest-example
2. Adding paths in the`tsconfig.json`:
```
...
"compilerOptions": {
...
    "paths": {
      "@src/*": ["src/*"]
    },
}
```
3. Modifying imports in `main.ts`
```
import { AppModule } from '@src/app.module';
```
4. Running "yarn start"

## Environment

![image](https://user-images.githubusercontent.com/23229557/71554838-8cc1eb00-2a5f-11ea-80cd-0dffd453b9c4.png)

**Comment by iWinston** [comment]
It causes by sequential path segment separation characters on Windows when running `lib/compiler/hooks/tsconfig-paths.hook.ts`:

```
function getNotAliasedPath(
  sf: ts.SourceFile,
  matcher: tsPaths.MatchPath,
  text: string,
) {
  const result = matcher(text, undefined, undefined, ['.ts', '.js']);
  if (!result) {
    return;
  }

  // result will contains '\' on Windows, but sf.fileName not.
  const resolvedPath = posix.relative(dirname(sf.fileName), result) || './';
  return resolvedPath[0] === '.' ? resolvedPath : './' + resolvedPath;
}
```
 @kamilmysliwiec Should I make a PR to fix the bug by adding this：
```
...
if (os.platform() === "win32") {
  result = result.replace(/\\g/, "/")
}
...
```

**Comment by kamilmysliwiec** [comment]
Fixed in 6.13.1

**Comment by ardyfeb** [comment]
@kamilmysliwiec still have this problem on 7.1.1
